Task/Workflow/TSKWorkflow.h

```diff
@@ -142,13 +144,13 @@ extern NSString *const TSKWorkflowTaskKey;
 @property (nonatomic, strong, readonly) NSNotificationCenter *notificationCenter;
```

Comment:
Same with the uncommentable lines, but unless I'm missing something `name` (ln 125) and `delegate` (ln 128) should be nullable